src/main/java/StringAddCalculator.java

```diff
@@ -37,14 +36,29 @@ private static String[] getArrayDividedByDelimiter(String input) {
     }
 
     private static boolean isContainNegative(String[] tokens) {
-        return Arrays.stream(tokens)
-                .anyMatch(token -> Integer.parseInt(token) < 0);
+        return Arrays.stream(tokens).anyMatch(token -> Integer.parseInt(token) < 0);
     }
 
     private static int sum(String[] tokens) {
-        return Arrays.stream(tokens)
-                .mapToInt(Integer::parseInt)
-                .sum();
+        return Arrays.stream(tokens).mapToInt(Integer::parseInt).sum();
```

Comment:
한줄에 점 하나씩으로 처리한다면 더 보기 편할거 같아요.
아울러 tokens에 잘못된 값이 있으면 어떻게 할까요?
지금 위에서도 뭔가 체크하고 있는듯 한데, 굳이 여러번 parseInt를 할필요가 없어보여요. 한번만 하도록 하면 어떨까요?
```suggestion
        return Arrays.stream(tokens)
        .mapToInt(Integer::parseInt)
        .sum();
```